.pylintrc

```diff
@@ -13,7 +13,6 @@ enable=
     bad-super-call,
     consider-merging-isinstance,
     continue-in-finally,
-    dangerous-default-value,
```

Comment:
I think this is useful to keep.

Reply:
Is there a way to ignore one specific line where this error is occurring? I have been hit by this `dangerous-default-value` without being able to see which parameter it is corresponding to or referring to. Will do some further investigations to find out if there is a different way to resolve this problem.

Reply:
Have solved this now.